example/docs/conf.py

```diff
@@ -19,7 +19,7 @@
 # -- General configuration ---------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
-sys.path.append(os.path.abspath("../.."))
+sys.path.append(str(Path("../..").resolve()))
```

Comment:
I understand that this is an improvement, but why did you do this as part of the same PR?